### PR TITLE
The Clerical Module Strikes Back

### PR DESCRIPTION
**Pull request by Ben10083**
I had most modules stripped of most of their clerical tools. We have clerical module after all, and some of the tools given were questionable (Quik pay for medical module???) 

The pen is mightier than the sword.

### CODE REVIEW

**Comment by Ben10083** [comment]
!balance

**Comment by Ben10083** [comment]
!review

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -391,11 +382,6 @@ var/global/list/robot_modules = list(
 	modules += new /obj/item/extinguisher(src) // For navigating space and/or low grav, and just being useful.
 	modules += new /obj/item/device/flash(src) // Non-lethal tool that prevents any 'borg from going lethal on Crew so long as it's an option according to laws.
 	modules += new /obj/item/crowbar/robotic(src) // Base crowbar that all 'borgs should have access to.
-	modules += new /obj/item/pen/robopen(src)
```
Comment (mikomyazaki) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:394]: Pen is important for the Construction module to be able to name doors, perhaps?

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -460,11 +446,6 @@ var/global/list/robot_modules = list(
...
```
Comment (mikomyazaki) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:463]: Same here. This module might have to rebuild a door and name it.

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (Ben10083) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:394]: Good point, re-added pen to both modules.

Comment (Ben10083) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:394]: Should maint drones be given the pen as well?

**Review by mikomyazaki — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**